src/sentry/static/sentry/app/actionCreators/navigation.jsx

```diff
@@ -18,6 +20,7 @@ export function navigateTo(to, router) {
           nextPath={to}
           needOrg={needOrg}
           needProject={needProject}
+          comingFromPojectId={comingFromPojectId}
```

Comment:
I think you're missing an `r` in Project.

Reply:
thanks, fixed